physical/raft/raft.go

```diff
@@ -1573,3 +1581,21 @@ func (s sealer) Open(ctx context.Context, ct []byte) ([]byte, error) {
 
 	return s.access.Decrypt(ctx, &eblob, nil)
 }
+
+// freelistOptions returns the freelist type and nofreelistsync values to use
+// when opening boltdb files, based on our preferred defaults, and the possible
+// presence of overriding environment variables.
+func freelistOptions() (bolt.FreelistType, bool) {
+	freelistType := bolt.FreelistMapType
+	noFreelistSync := true
+
+	if os.Getenv("VAULT_FREELIST_TYPE") == "array" {
+		freelistType = bolt.FreelistArrayType
+	}
+
+	if os.Getenv("VAULT_NO_FREELIST_SYNC") == "false" {
```

Comment:
We commonly accept any non-empty value for a boolean as "true".

Reply:
But in this case, since the default is `true`, I need some way of ensuring that the user explicitly wants to turn this off. `false` felt like the obvious choice in this case, but is there something else I should use?

Reply:
Oh, heh, right.  Why don't we avoid the confusion of double negatives, and just say if you have a non-empty `VAULT_FREELIST_SYNC` env var, we'll turn freelist syncing back on, i.e. set `noFreelistSync = false`?

As a side note, I wonder if we want to add `BOLTDB` to these env vars to make it clearer to someone who's not familiar with this code what they're about, or even `RAFT` or `RAFT_BOLTDB`?  We do have a couple of env vars already that start with `VAULT_RAFT`.

Reply:
Sure, `VAULT_RAFT` is fine.